app/queries/explorer_stake_account_query.rb

```diff
@@ -12,9 +14,16 @@ def initialize(withdrawer: nil, staker: nil, vote_account: nil, stake_pubkey: ni
   def call(page: 1, per: 20)
     explorer_stake_accounts = ExplorerStakeAccount.where(
       network: @network,
-      epoch: EpochWallClock.by_network(@network).first&.epoch
+      epoch: EpochWallClock.by_network(@network).last&.epoch
     )
 
+    if explorer_stake_accounts.count < MIN_ACCOUNTS_NUMBER
```

Comment:
`EpochWallClock.by_network(@network).last` can be nil? If so then `count` method will raise an error.